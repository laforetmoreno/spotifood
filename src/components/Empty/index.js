import React from 'react';
import { string } from 'prop-types';

const Empty = ({ message, className }) => <p className={className}>{message}</p>;

Empty.propTypes = {
  message: string,
  className: string,
};

Empty.defaultProps = {
  message: 'Ops, no results.',
  className: '',
};

export default Empty;
