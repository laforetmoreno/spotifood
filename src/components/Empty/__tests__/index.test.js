import React from 'react';
import renderer from 'react-test-renderer';
import Empty from '../index';

describe('<Empty />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Empty />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
