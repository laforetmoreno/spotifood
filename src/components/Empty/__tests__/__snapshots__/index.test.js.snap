// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Empty /> renders correctly 1`] = `
<p
  className=""
>
  Ops, no results.
</p>
`;
